app/apps/server/bridges/messages.ts

```diff
@@ -50,6 +50,11 @@ export class AppMessageBridge extends MessageBridge {
 		updateMessage(msg, editor);
 	}
 
+	protected async delete(messageId: string, appId: string): Promise<void> {
+		this.orch.debugLog(`The App ${appId} is deleting a message.`);
+		Messages.removeById(messageId);
```

Comment:
We cannot delete the database entry directly from here. There are a couple of checks that need to be performed before we actually delete a message from DB. You can take a look at this file to understand more about this: `app/lib/server/functions/deleteMessage.ts`.